clis/teliod/src/main.rs

```diff
@@ -129,19 +207,193 @@ impl CommandListener {
     }
 }
 
+async fn init_api(
+    config: &mut TeliodDaemonConfig,
+    config_path: String,
+) -> Result<ClientConfig, TeliodError> {
+    let mut client_config = ClientConfig::new(config).await?;
+
+    let mut retries = 0;
+    loop {
+        let status = update_machine(&client_config).await?;
```

Comment:
It seems that on the first run of this loop machine id is "", so you just make update request on incorrect URL, what is the point of that? You can have machine id in config, but it would be nice to check it before using it.

Reply:
But machine id is in config, and if it is there, it is loaded here https://github.com/NordSecurity/libtelio/pull/853/files/d07eb05a2b9994218dec09da44e790f0461d355f#diff-adacd55ecee4795f10ea1a14eea3906b343e77c9c9d8a717761e1f82f2f2e7faR375

Reply:
Yeah, but it might not be there

Reply:
And also machine ID feels to me like a bit internal thing - I'm not sure if we should bother end user with it (soo, I'm not sure if it really should be in config)

Reply:
Yes. But that is how it is supposed to work.
And regarding machine ID being in config, I mean, the end user isn't really bothered with it, because it would download it if not present, and if it is there then 🤷